packages/cli/src/commands/update.ts

```diff
@@ -0,0 +1,78 @@
+import fs from "fs";
+import path from "path";
+import fetch from "node-fetch";
+import inquirer from "inquirer";
+import { installDependencies } from "../utils/installDependencies.js";
+
+export async function updateCommand(projectPath: string) {
+  const packageJsonPath = path.join(projectPath, "package.json");
+
+  if (!fs.existsSync(packageJsonPath)) {
+    console.error("package.json not found in the current directory.");
+    return;
+  }
+
+  const packageJsonData = JSON.parse(fs.readFileSync(packageJsonPath, "utf8"));
+  const dependencies = packageJsonData.dependencies || {};
+  const devDependencies = packageJsonData.devDependencies || {};
+
+  const triggerPackages = Object.keys({ ...dependencies, ...devDependencies }).filter(
+    (packageName) => packageName.startsWith("@trigger.dev/")
+  );
+
+  if (triggerPackages.length === 0) {
+    console.log("No @trigger.dev/* packages found in package.json.");
+    return;
+  }
+
+  const newVersions = await Promise.all(
+    triggerPackages.map(async (packageName) => {
+      try {
+        const response = await fetch(`https://registry.npmjs.org/${packageName}`);
+        const data = await response.json();
+        const latestVersion = data["dist-tags"].latest;
+        return { packageName, latestVersion };
+      } catch (error) {
+        // @ts-ignore
+        console.error(`Error fetching version for ${packageName}: ${error.message}`);
+        return null;
+      }
+    })
+  );
+  const packagesToUpdate = newVersions.filter(
+    (entry) => entry !== null && entry.latestVersion !== dependencies[entry.packageName]
+  );
+
+  if (packagesToUpdate.length === 0) {
+    console.log("All @trigger.dev/* packages are up to date.");
+    return;
+  }
+
+  console.log("Newer versions found for the following packages:");
+  packagesToUpdate.forEach((entry) => {
+    console.log(
+      `- ${entry.packageName}: current ${dependencies[entry.packageName]} -> latest ${
```

Comment:
There are TypeScript errors here because entry can be null. The easiest fix would be to do a null check before line 53.